easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.2.2-foss-2017b-Perl-5.26.0.eb

```diff
@@ -0,0 +1,48 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2017 University of Geneva
+# Authors::   Yann Sagon <yann.sagon@unige.ch>
+# License::   MIT/GPL
+# $Id$
+#
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'MaSuRCA'
+version = '3.2.2'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'http://www.genome.umd.edu/masurca.html'
+
+description = '''MaSuRCA is whole genome assembly software. It combines the efficiency of the de Bruijn graph  
+ and Overlap-Layout-Consensus (OLC) approaches. MaSuRCA can assemble data sets containing 
+ only short reads from Illumina sequencing or a mixture of short reads and long reads 
+ (Sanger, 454, Pacbio and Nanopore).'''
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+# need a temporary url to download it. Do it manually here:
+# http://www.genome.umd.edu/masurca_form.html
```

Comment:
is `3.2.2` version still easily available? From `3.2.4` it seems to be on github (https://github.com/alekseyzimin/masurca/releases), perhaps bump version and fetch from there?

Reply:
@migueldiascosta I needed that specific version, requested by an user. Bumping helps to get easyer the source, but then that should be a separate PR.

Reply:
@hajgato ok - can you submit a test report for `3.2.2` then?